blockbuilder/genesisState.go

```diff
@@ -0,0 +1,50 @@
+package blockbuilder
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/NethermindEth/juno/core"
+	"github.com/NethermindEth/juno/core/felt"
+)
+
+var ErrChainIDRequired = errors.New("ChainID is required")
+
+type GenesisConfig struct {
+	ChainID                 string           `json:"chain_id"`
+	WhitelistedSequencerSet []felt.Felt      `json:"whitelisted_sequencer_set"`
+	State                   genesisStateData `json:"state"`
+}
+
+type genesisStateData struct {
+	Classes       map[string]genesisClassData    `json:"classes"`
+	Contracts     map[string]genesisContractData `json:"contracts"`
+	Storage       map[string][]core.StorageDiff  `json:"storage"`
```

Comment:
I don't think we have an immediate use for this when we have `FunctionCalls`

Reply:
Removed the Storage field